src/main/java/core/basesyntax/serviceimpl/HandlerReceiverImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.operations.BalanceOperation;
+import core.basesyntax.operations.OperationHandler;
+import core.basesyntax.operations.PurchaseOperation;
+import core.basesyntax.operations.ReturnOperation;
+import core.basesyntax.operations.SupplyOperation;
+import core.basesyntax.service.HandlerReceiver;
+
+public class HandlerReceiverImpl implements HandlerReceiver {
+    @Override
+    public OperationHandler getHandler(String operation) {
+        switch (operation) {
+            case "b":
+                return new BalanceOperation();
+            case "s":
+                return new SupplyOperation();
+            case "r":
+                return new ReturnOperation();
+            case "p":
+                return new PurchaseOperation();
+            default:
+                throw new RuntimeException("Invalid operation " + operation);
+        }
```

Comment:
Use map instead of switch 
![image](https://github.com/mate-academy/jv-fruit-shop/assets/32339836/21ee1459-ae66-44f5-8e0f-0438f3d04a7e)